From 47ca82473af8d77eb89732884542c719ccdd9312 Mon Sep 17 00:00:00 2001
From: Stefan Agner <stefan@agner.ch>
Date: Thu, 29 Feb 2024 19:07:15 +0100
Subject: [PATCH] Linux: Increase number of endpoints

Users run into endpoint limits, posting "Endpoint pool full" in various
setups. This is likely caused by installations with many containers
running or similar.

Quadruple the amount of endpoints supported to make it less likely to
appear.
---
 src/platform/Linux/InetPlatformConfig.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/platform/Linux/InetPlatformConfig.h b/src/platform/Linux/InetPlatformConfig.h
index 3aab9a7b9b..02e664eddc 100644
--- a/src/platform/Linux/InetPlatformConfig.h
+++ b/src/platform/Linux/InetPlatformConfig.h
@@ -33,7 +33,7 @@
 // ========== Platform-specific Configuration Overrides =========
 
 #ifndef INET_CONFIG_NUM_TCP_ENDPOINTS
-#define INET_CONFIG_NUM_TCP_ENDPOINTS 32
+#define INET_CONFIG_NUM_TCP_ENDPOINTS 128
 #endif // INET_CONFIG_NUM_TCP_ENDPOINTS
 
 #ifndef IPV6_MULTICAST_IMPLEMENTED
@@ -41,7 +41,7 @@
 #endif
 
 #ifndef INET_CONFIG_NUM_UDP_ENDPOINTS
-#define INET_CONFIG_NUM_UDP_ENDPOINTS 32
+#define INET_CONFIG_NUM_UDP_ENDPOINTS 128
 #endif // INET_CONFIG_NUM_UDP_ENDPOINTS
 
 // On linux platform, we have sys/socket.h, so HAVE_SO_BINDTODEVICE should be set to 1
-- 
2.44.0

